### PR TITLE
feat:map added text made the map and used linear grarient to style th…

### PR DESCRIPTION
**Pull request by ell-sie**
…e div infront of the map

# Description
- [x] Added the "Map" section.
- [x] Styled the section using appropriate classes for layout, fonts, position, and colors.

## Motivation and Context
The added content provides a map of Rwanda. The styling has been applied to make the section visually appealing.

## How Has This Been Tested, and How are your changes run?
- [x] Checked the content for accuracy and correctness.
- [x] Ensured that the text was properly aligned and visually appealing.
- [x] Verified that the buttons and links are functional.

## Screenshots (if appropriate):
image
image

### CODE REVIEW

**Review by peninah98 — APPROVED** [review]

File: src/pages/map.html [hunk 1]
```diff
@@ -1,6 +1,36 @@
 <div class="">
     <div class="h-28 -mt-28" id="curve-to-start-Qualitat-section" style="background: url(./assets/BGs/qualitat.svg); background-position: top; background-size: cover;"></div>
-    <div class="text-white bg-[#001342]" id="rwanda-map-section-description">Rwanda map section description</div>
+    <div class="text-white bg-[#001342] flex flex-col gap-5 px-24">
```
Comment (alainmigabo) [inline src/pages/map.html:3]: You need to remove this arbitrary value. If you can't find this color you can define it in the tailwind.config.js file

File: src/pages/map.html [hunk 2]
```diff
@@ -1,6 +1,36 @@
...
+          <div class="absolute top-[385px] left-[637px] flex gap-3">
+            <img src="../../assets/images/image.svg"  alt="">
+          <div class=" flex flex-col">
+            <h2 class="font-bold w-full relative font-montserrat">Ruanda<div class="h-1 bg-green-primary-trans w-16 absolute top-[18px] -left-0 -rotate-3"></div></h2>
+            <p class="font-montserrat text-xs font-light">Hauptstadt:Kigali</p>
+          </div>
+          </div>
+          
+        </div>
+        <div class=" absolute bg-gradient-to-r from-green-primary-trans/50 to-transparent to-[60%]  mt-[250px] w-3/4 p-16 px-32 flex flex-col gap-8">
```
Comment (alainmigabo) [inline src/pages/map.html:22]: Here too you need to remove this arbitrary value. Use '3/5' instead of '[60%]'

**Review by alainmigabo — CHANGES_REQUESTED** [review]

File: src/pages/map.html (hunk 1 above)
Comment (ell-sie) [reply src/pages/map.html:3]: done

File: src/pages/map.html (hunk 2 above)
Comment (ell-sie) [reply src/pages/map.html:22]: 3/5 throws an error it only accepts percentages.

**Review by Nezerwa — APPROVED** [review]
Congratulations @ell-sie You task is complent now , it's time to merge

File: src/pages/map.html (hunk 2 above)
Comment (alainmigabo) [reply src/pages/map.html:22]: okay

**Review by alainmigabo — APPROVED** [review]
Now let's see if we don't have any conflicts

**Pull request merged**